src/main/java/org/gridsuite/explore/server/services/DirectoryService.java

```diff
@@ -27,6 +27,7 @@
 
 import static org.gridsuite.explore.server.ExploreException.Type.UNKNOWN_ELEMENT_TYPE;
 import static org.gridsuite.explore.server.services.ExploreService.*;
+import static org.gridsuite.explore.server.services.NotificationType.UPDATE_DIRECTORY;
```

Comment:
Remove static import and use NotificationType.UPDATE_DIRECTORY in the code

Reply:
Done